27596 UUID fix (#3615)
demisto/demisto-sdk: CHANGELOG.md

* Updated the **format** command to use the content graph instead of the id_set file.
* Updated **format** command not to fail on unexpected values that returns from the graph, and just add it to the log.
* Removed a redundant debug log on the `tools.get_file` function.

> * Adding a new boolean flag *auto-replace-uuids* to **download** command.

Reviewer: After changing the flag name, make sure to add a description about the flag here:
Use this flag to avoid UUID replacements when downloading (which content items it affects) using **download** command.

Reviewer 2: ```suggestion
* Added the *auto-replace-uuids* flag to the **download** command. Use it to ... <what Yuval suggested>
```

> * Added the *auto-replace-uuids* flag to the **download** command. Use it to ... <what Yuval suggested>

Reviewer 2: `<...>` 🥲

Author: After applying the [suggested change](https://github.com/demisto/demisto-sdk/pull/3615#discussion_r1321108070).
I was fixed a commit after

> * Added the *auto-replace-uuids* flag to the **download** command. Use it to avoid UUID replacements when downloading using download command.

Reviewer: ```suggestion
* Added the *auto-replace-uuids* flag to the **download** command. set this flag to False to avoid UUID replacements when downloading using download command.
```

* Updated the **format** command to use the content graph instead of the id_set file.
* Updated **format** command not to fail on unexpected values that returns from the graph, and just add it to the log.
* Removed a redundant debug log on the `tools.get_file` function.

> * Added the *auto-replace-uuids* flag to the **download** command. set this flag to False to avoid UUID replacements when downloading using download command.

Reviewer 3: please update changelog location
